dashboard/test/ui/step_definitions/netsim.rb

```diff
@@ -15,6 +15,7 @@
 
 When /^I enter the netsim name "([^"]*)"$/ do |username|
   steps %{
+    And I wait until element "#netsim-lobby-name" is visible
     And I type "#{username}" into "#netsim-lobby-name"
```

Comment:
Instead, should this step implicitly wait until the element is visible before typing text?